testing/src/test/java/org/motechproject/nms/testing/it/kilkari/MctsBeneficiaryUpdateServiceBundleIT.java

```diff
@@ -521,7 +521,7 @@ public void testUpdateBeneficiariesFromFile() throws Exception {
         assertEquals(453L, (long) updatedMother3.getHealthBlock().getCode());
 
         // DOB update:
-        String updatedDOB = "01-07-2015";
+        String updatedDOB = "01-12-2015";
```

Comment:
Is there a reason this date had to be changed?  Will we run into a time specific failure in the future or something?